app/controllers/users_controller.rb

```diff
@@ -18,7 +18,7 @@ def update
     set_user
     set_tabs(params["user"]["tab"])
     if @user.update(permitted_attributes(@user))
-      RssReader.new.delay.fetch_user(@user) if @user.feed_url.present?
+      RssReaderFetchUserJob.perform_later(@user.id) if @user.feed_url.present?
```

Comment:
I wonder if the condition `if @user.feed_url.present?` shouldn't be moved inside the job

For two reasons: 
- if it's called in another part of the app, we have to remember to check the same condition
- if between the scheduling and execution the feed_url gets removed, the job will fail

what do you think @lightalloy ?

Reply:
Good point, the second reason seems especially strong to me.

Reply:
I have updated the job in that way.